lib/ecs_rails_console/core.rb

```diff
@@ -36,12 +36,12 @@ def subnet_ids(vpc_id)
         ids = ec2_client.describe_subnets(
           {
             filters: [
-              { name: 'vpc-id', values: [vpc_id] },
-              { name: 'tag:aws-cdk:subnet-type', values: ['Public'] }
+              {name: "vpc-id", values: [vpc_id]},
+              {name: "tag:aws-cdk:subnet-type", values: ["Public"]}
```

Comment:
I personally prefer space between curly brackets, it looks cleaner. Also, single quote is faster to type, I don't need to press 2 buttons (shift + quote key).

Reply:
Standard uses spaces inside blocks, but not hash literals - In Ruby, the { and } characters do a lot of heavy lifting. To visually distinguish hash literals from blocks, Standard enforces that (like arrays), no leading or trailing spaces be added to pad hashes

Reply:
Double quotes for string literals - because pre-committing to whether you'll need interpolation in a string slows people down

Reply:
I think the purpose of the standard is to take out the personal preference, I like space in hash literals too.